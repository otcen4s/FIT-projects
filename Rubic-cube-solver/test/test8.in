343
113
112
441 521 232 633
521 232 644 551
522 666 433 441
455
665
665