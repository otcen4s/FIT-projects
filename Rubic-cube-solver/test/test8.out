343
113
112
441 521 232 633
521 232 644 551
522 666 433 441
455
665
665

113
114
233
521 232 633 441
521 232 644 551
522 666 433 441
455
665
665

111
111
232
525 622 333 441
525 633 444 551
525 622 333 441
454
666
666

111
111
111
555 222 333 444
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

